org.gimp.GIMP.json

```diff
@@ -691,22 +691,14 @@
             "sources": [
                 {
                     "type": "archive",
-                    "url": "https://gitlab.freedesktop.org/cairo/cairo/-/archive/1.17.4/cairo-1.17.4.tar.gz",
-                    "sha256": "0a673c88f7c049b4555c1f152b11d9fe16d8523b62632d2d5c14e1868499ce9f",
+                    "url": "https://gitlab.freedesktop.org/cairo/cairo/-/archive/1.17.6/cairo-1.17.6.tar.gz",
+                    "sha256": "a2227afc15e616657341c42af9830c937c3a6bfa63661074eabef13600e8936f",
                     "x-checker-data": {
                         "type": "anitya",
                         "project-id": 247,
                         "stable-only": true,
                         "url-template": "https://gitlab.freedesktop.org/cairo/cairo/-/archive/$version/cairo-$version.tar.gz"
                     }
-                },
-                {
-                    "type": "patch",
-                    "path": "patches/cairo-mr114-gimp-issue-6210.patch"
```

Comment:
these two patches haven't been removed from the repository. They should be.

Reply:
Good catch! Thanks!